src/detections/table/body/hooks/use-row-props.js

```diff
@@ -0,0 +1,11 @@
+import { useMediaQuery, useTheme } from '@material-ui/core'
+
+export const useRowProps = hideOnMobile => {
```

Comment:
Let's rename this hook to `useCellProps`.

Reply:
Let's make this parameter to be an complex object, as that will make it easier for us to extend it later on and be more readable.

So, our hook's new signature would be:
```js
export const useCellProps = ({hideOnMobile}) => ...
```

And we'd use it like this:
```js
<TableCell {...useCellProps({hideOnMobile: true})} />
```

Reply:
Done